types/force-graph/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for force-graph 1.15
+// Type definitions for force-graph 1.21
 // Project: https://github.com/vasturiano/force-graph
 // Definitions by: Peter Kimberley <https://github.com/p-kimberley>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+//                 Noah Santschi-Cooney <https://github.com/Strum355>
```

Comment:
More than one contributor. Should be in format as defined at: https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package:

// Definitions by: Peter Kimberley <https://github.com/p-kimberley>
//                 Noah Santschi-Cooney <https://github.com/Strum355>

Reply:
Yup that was an oversight, fixed :)